include/aspect/mesh_refinement/volume_of_fluid_interface.h

```diff
@@ -0,0 +1,75 @@
+/*
+  Copyright (C) 2013 by the authors of the ASPECT code.
+
+  This file is part of ASPECT.
+
+  ASPECT is free software; you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation; either version 2, or (at your option)
+  any later version.
+
+  ASPECT is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with ASPECT; see the file doc/COPYING.  If not see
+  <http://www.gnu.org/licenses/>.
+*/
+
+
+#ifndef __aspect_mesh_refinement_volume_of_fluid_interface_h
+#define __aspect_mesh_refinement_volume_of_fluid_interface_h
```

Comment:
Please use single underscores for all include guards in all header files (we removed the double underscores, as they are reserved for compiler purposes).